tests/perfs/perfs.py

```diff
@@ -25,11 +25,14 @@ def load_config():
 
 conf = load_config()
 
-def execute(suit, bin_path, host, port, output_dir):
+def execute(suit, bin_path, host, port, concurrency, iteration, output_dir):
     base_cfg = conf['config']
-
-    iterations = suit.get("iterations", base_cfg['iterations'])
-    concurrency = suit.get("concurrency", base_cfg['concurrency'])
+    if iteration == "" :
+        iterations = suit.get("iterations", base_cfg['iterations'])
+    else :
+        iterations = iteration
```

Comment:
Seems single variable `iterations`  is enough like `concurrency`.

Reply:
agreed and updated